### PR TITLE
fix: add gulp dependency for inline html

### PR DESCRIPTION
**Pull request by JAHMCR**
#84 

Add `gulp` dependency for inline html inside the minified js

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ibm-aspera/connect-sdk-js",
-  "version": "5.0.1",
+  "version": "5.0.2",
```
Comment (dwosk) [inline package.json:3]: I forget - do we need to bump the version manually? I thought it was done automatically at release time in CI.

**Review by dwosk — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (JAHMCR) [reply package.json:3]: I had to bump it manually with 5.0.1 [here](https://github.com/IBM/aspera-connect-sdk-js/pull/73)

**Review by dwosk — CHANGES_REQUESTED** [review]
Updating the PR based on our observations that the JS and CSS are still not being inlined.

**Review by dwosk — APPROVED** [review]
Thanks!

**Pull request merged**